src/3d/chunks/qgschunkedentity_p.h

```diff
@@ -194,6 +194,7 @@ class QgsChunkedEntity : public Qt3DCore::QEntity
     bool mIsValid = true;
 
     int mPrimitivesBudget = std::numeric_limits<int>::max();
+    long long mGpuMemoryLimit = std::numeric_limits<long long>::max();
```

Comment:
I would prefer to keep the GPU memory limit always in MB, not in bytes - it will make things simpler and we can use just `int` instead of `long long` for the size (and no recalculations 1024*1024 in the GUI)